src/vsr/replica.zig

```diff
@@ -1682,7 +1685,7 @@ pub fn Replica(
 
         /// Returns whether `b` succeeds `a` by having a newer view or same view and newer op.
         fn ascending_viewstamps(
-            self: *Self,
+            _: *Self,
```

Comment:
We could just remove this argument instead, which would make the simple nature of this function clearer at the callsite.